src/components/CheckResultsByLevel.tsx

```diff
@@ -41,7 +42,7 @@ const useStyles = makeStyles((theme: BackstageTheme) => {
   };
 });
 
-export function CheckResultsByLevel({ checkResultsByLevel, totalChecks, totalPassingChecks }: Props) {
+export function CheckResultsByLevel({ opslevelUrl, checkResultsByLevel, totalChecks, totalPassingChecks }: Props) {
```

Comment:
Nitpick - I know this is time-sensitive, but is this something we can [pull out of config](https://backstage.io/docs/conf/reading#accessing-configapi-in-frontend-plugins) instead of having to remember to pass this as a prop?

Reply:
Ah, that's way better. I've been flying on this stuff and have not read about backstage at all.

Reply:
IMO worth doing, but worth doing later

Reply:
probably worth doing now, the generated URLs are currently wrong :(